docs/joss/paper.bib

```diff
@@ -16,15 +16,15 @@ @misc{g95-site
 
 @misc{ifx-site,
   author={{Intel Corporation}},
-  title = {{Intel Fortran Compiler}},
+  title = {Intel {F}ortran Compiler},
```

Comment:
Similarly, [Intel Corporation literature](https://www.intel.com/content/www/us/en/developer/tools/oneapi/fortran-compiler.html) capitalizes the name of their Intel Fortran Compiler product, even outside of titles:

```suggestion
  title = {{Intel Fortran Compiler}},
```